### DISCUSSION TITLE
Table Editor URLs of paused projects cause endless loading

### DISCUSSION THREAD

**Discussion by hyunbinseo** [supabase/supabase#16929] [Contribute to Supabase]
# Bug report

## Describe the bug

(Dashboard) Directly accessing the Table Editor in a paused project shows an endless loading spinner.

## To Reproduce

- Prepare a paused project
- Directly access the Table Editor's specific table with an URL.

```
https://app.supabase.com/project/{project-id}/editor/{table-id}
```

- The loading spinner keeps on spinning with no feedback.
- Move to the Home tab using the left sidebar navigation.
- Successfully shows the following information.

> This project is paused.
> Restore this project and get back to building the next big thing!

## Expected behavior

All child routes should show that the project is paused.

## Additional Context

The following route does show an error message after a few seconds.

```
https://app.supabase.com/project/{project-id}/editor
```

> Error: [500] An error has occurred: Failed to fetch

**Comment by saltcod** [comment]
Hey @hyunbinseo is this still occurring for you?
